packages/debugger/lib/data/actions/index.js

```diff
@@ -99,3 +99,20 @@ export function allocate(
     state
   };
 }
+
+export const SET_STORAGE_VISIBILITY = "DATA_SET_STORAGE_VISIBILITY";
```

Comment:
"storage visibility" is a weird name... wouldn't this be better named something like "storage fetching" or "intertransactional storage" or something?

Reply:
Hm, yeah, I see where you're coming from. I just don't think it'll make as much sense to someone trying to make sense of the code, vs. something that says more specifically what it relates to.

Along those lines, I think "storage fetching" might not even be sufficient, because that might lead one to believe it's `eth_getStorageAt`, and so I'd want to discriminate the contrast there as best we can. But "intertransactional storage access" is a mouthful (or maybe not, I kinda like it?)

Reply:
or maybe like... "precision storage lookup"

I feel like we need to decide what we will be calling this feature, publicly speaking, and go with that as the basis of our naming.

Reply:
(After discussing this more with @gnidan, we decided to go with "storage lookup", and to rename `fetchStorage` to `storageLookup`.)